Formula/pianod.rb

```diff
@@ -20,15 +24,20 @@ class Pianod < Formula
   depends_on "libao"
   depends_on "libgcrypt"
 
+  on_macos do
+    depends_on "ncurses"
+  end
+
   on_linux do
     # pianod uses avfoundation on macOS, ffmpeg on Linux
     depends_on "ffmpeg"
+    depends_on "gcc"
     depends_on "gnutls"
     depends_on "libbsd"
   end
 
```

Comment:
```suggestion

  fails_with gcc: "5"

```
We need this to actually use Homebrew GCC for the build.